client/landing/stepper/declarative-flow/internals/components/step-route/test/index.tsx

```diff
@@ -0,0 +1,182 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { waitFor } from '@testing-library/react';
+import { addQueryArgs } from '@wordpress/url';
+import React, { FC, Suspense } from 'react';
+import { MemoryRouter } from 'react-router';
+import recordStepStart from 'calypso/landing/stepper/declarative-flow/internals/analytics/record-step-start';
+import {
+	Flow,
+	StepperStep,
+	StepProps,
+} from 'calypso/landing/stepper/declarative-flow/internals/types';
+import { useIntent } from 'calypso/landing/stepper/hooks/use-intent';
+import { useSelectedDesign } from 'calypso/landing/stepper/hooks/use-selected-design';
+import { recordPageView } from 'calypso/lib/analytics/page-view';
+import {
+	getSignupCompleteFlowNameAndClear,
+	getSignupCompleteStepNameAndClear,
+} from 'calypso/signup/storageUtils';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
+import StepRoute from '../';
+import { NavigationControls } from '../../../types';
```

Comment:
Nit: This is another type.
```suggestion
import type { NavigationControls } from '../../../types';
```

Reply:
All missing 'type' fixed here 5957f4e698ebdca226b315732e85a21c72da1b06